sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/BeamSqlCli.java

```diff
@@ -78,45 +67,15 @@ public void execute(String sqlString) throws Exception {
     BeamSqlParser parser = new BeamSqlParser(sqlString);
     SqlNode sqlNode = parser.impl().parseSqlStmtEof();
 
-    if (sqlNode instanceof SqlCreateTable) {
-      handleCreateTable((SqlCreateTable) sqlNode, metaStore);
-    } else if (sqlNode instanceof SqlDropTable) {
-      handleDropTable((SqlDropTable) sqlNode);
+    if (sqlNode instanceof SqlExecutableStatement) {
+      ((SqlExecutableStatement) sqlNode).execute(env.getContext());
     } else {
```

Comment:
nit: add a comment what is executable statement, what is not?

Reply:
Comment added: DDL nodes are SqlExecutableStatement